go.mod

```diff
@@ -4,86 +4,84 @@ go 1.15
 
```

Comment:
k8s 1.21 is built on go1.16.4, should we bump go to 1.16 here as well?

Reply:
Good point. I was planning to do it in a separate PR, as we will also need to update the go version in docker files.

https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/8daee204ea7d9ba8ea782c72f2b8cb8157362d45/images/driver/Dockerfile#L19
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/8daee204ea7d9ba8ea782c72f2b8cb8157362d45/images/syncer/Dockerfile#L17

Reply:
/approve
ok, when you are submitting the follow up PR, give QE a heads up, we need to bump up Go version in jenkins workers

Reply:
Sure I will do that.